engines/python/setup/djl_python/tests/test_lmi_dist.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+#
+# Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+# except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+import unittest
+
+import torch
+import os
+import sys
+
+
+class TestLmiDist(unittest.TestCase):
+
+    @unittest.skipIf(not torch.cuda.is_available(), "CUDA is not available")
+    def test_models(self):
+        # === Preparation ===
+        script_directory = os.path.dirname(os.path.abspath(__file__))
+        relative_path = "../../"
+        new_path = os.path.normpath(
+            os.path.join(script_directory, relative_path))
+        sys.path.append(new_path)
+        sys.path.append("/usr/local/lib/python3.9/dist-packages/lmi_dist")
+        from djl_python.rolling_batch.lmi_dist_rolling_batch import LmiDistRollingBatch
+        from djl_python.tests.rolling_batch_test_scripts.generator import Generator, print_rank0
+
+        # --- Models ---
+        model_names = [
+            # "TheBloke/Llama-2-7B-Chat-fp16",
+            # TODO: fix this. weight model.layers.0.self_attn.rotary_emb.inv_freq does not exist
+            # "TheBloke/Llama-2-7B-Chat-AWQ",
+            "TinyLlama/TinyLlama-1.1B-Chat-v0.6",
+            # TODO: fix this. weight model.layers.0.self_attn.rotary_emb.inv_freq does not exist
+            # "TinyLlama/TinyLlama-1.1B-python-v0.1",
+            # g5.12xlarge single gpu ok. But no way to clear the gpu memory after running llama-2-7b thus cause OOM
+            # "codellama/CodeLlama-7b-hf"
+        ]
+        expected_text_30 = {
+            "TheBloke/Llama-2-7B-Chat-fp16": {
```

Comment:
let's remove this then

Reply:
But it is just too big to test on a CPU machine. We could use a better or smaller model for evaluation and testing